### DISCUSSION TITLE
<podcast:season> tag - add image attribute

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#638] [Enhancement Proposal]
Add an image href attribute to the season tag for season level art.

**Comment by nathangathright** [comment]
I presume the recommended UX would be that apps display this artwork for episodes with a matching season number/name superseding show-level artwork, and this image would itself be superseded by episode-level artwork? Is there a `medium="music"` angle that should be documented?

Podcasts like [Land of the Giants](https://podcastindex.org/podcast/573729) have worked around this problem by including the season-level artwork within each item as the episode-level artwork, and I imagine they’d continue for maximum compatibility across apps. So, I fear the images in this attribute will never see the light of day.

I’d love to see apps take advantage of images in the `podcast:season` tag to promote full season of shows, but that would require a critical mass of adoption before it becomes viable. When a tag/attribute needs to cross that threshold before it provides any marginal utility, it has very little chance of gaining momentum.

**Comment by thomasrynne** [comment]
Before proceeding I think it is worth re-visiting @mitchdowney 's [comment](https://github.com/Podcastindex-org/podcast-namespace/issues/108#issuecomment-726811299) in the original season tag discussion (I wasn't around then, I just found it when reading the history of the season tag).

I agree that season should probably have been a channel level tag rather than an item level tag.

At the moment the season name needs to be repeated for every item, and there is ambiguity if items in the same season have different values for season name.

Adding image makes this worse and it seems likely that 'season note/description' could be added in the future.

In database terms: season information is being added to the item table instead of a referencing a season table.

Is it too late to add season as a channel level item?

**Reply by theDanielJLewis** [reply]
Here's a way we could solve that.

Put a `season` tag in each episode as we currently do, because each episode needs that (for seasonal shows). Then have a sort of season index at the channel level and _that_ is where the numbers are given titles and images.

For example:

```xml
<podcast:season number="4" name="Race for the White House 2024" image="IMAGE-URL" />
<podcast:season number="3.5" name="2022 Midterms" image="IMAGE-URL" />
<podcast:season number="3" name="Race for the White House 2020" image="IMAGE-URL" />
<podcast:season number="2" name="Race for the White House 2016" image="IMAGE-URL" />
```

With that sort of "index," individual episodes could use simply the season number, like:

```xml
<podcast:season>3.5</podcast:season>
```

**Reply by thomasrynne** [reply]
Yes, I think this is better. However, it might be confusing that valid season tag attributes depend on the placement of the tag.
